README.md

```diff
@@ -1,16 +1,31 @@
-# Caution ⚠️
+# IceDOS
 
+This is a NixOS project that aims to create a highly opinionated gaming and general purpose computing experience, while still providing enough configuration options for my different needs.
```

Comment:
I would like this to be described better as a configuration project.

Reply:
Wasn't sure how to prevent confusion with the usage of the word "configuration" multiple times, but I think I struck a good balance.